services/ui-src/src/components/forms/AdminDashSelector.tsx

```diff
@@ -30,11 +30,21 @@ export const AdminDashSelector = ({ verbiage }: Props) => {
     id: "MLR",
     label: "Medicaid Medical Loss Ratio (MLR)",
   };
+  const naaarReportChoice = {
+    id: "NAAAR",
+    label: "Network Adequacy and Access Assurances Report (NAAAR)",
+  };
 
   // assemble and inject report choices depending on whether report is enabled
   const mlrReport = useFlags()?.mlrReport;
+  const naaarReport = useFlags()?.naaarReport;
   const reportField = formJson.fields.find((field) => field.id === "report")!;
 
+  if (naaarReport) {
+    reportField.type = "radio";
+    reportField.validation = "radio";
+    reportChoices.push(naaarReportChoice);
+  }
```

Comment:
I noticed this button showed up between mcpar and mlr. Putting it at the end would keep things consistent alphabetically and with the state view